.github/workflows/generate-deb-packages.yaml

```
@@ -0,0 +1,20 @@
+name: generate-deb-packages
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  generate-deb-packages:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```
This currently only works for 22.04